x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/store/KibanaOwnedReservedRoleDescriptors.java

```diff
@@ -281,13 +281,12 @@ static RoleDescriptor kibanaSystem(String name) {
                 RoleDescriptor.IndicesPrivileges.builder()
                     .indices("logs-ti_*_latest.*")
                     .privileges(
-                        // Require "create_index", "delete_index", "read", "index", "delete", IndicesAliasesAction.NAME, and
-                        // UpdateSettingsAction.NAME for transform
                         "create_index",
                         "delete_index",
                         "read",
                         "index",
                         "delete",
+                        "manage",
```

Comment:
Can you provide us with a bit more context why the Kibana system user needs `manage` permissions for this index?

Reply:
Hey @thomheymann, for threat intel integrations using transforms, we would like to define destination indices aliases like this: https://github.com/elastic/integrations/pull/7530/files#diff-0217ebe0b8d79ca1051565d04c59a24a771a213dbd9f289981d59fd35c605e42R13-R15 which has `move_on_creation: true` in order to [maintain single destination index for that alias](https://www.elastic.co/guide/en/elasticsearch/reference/current/put-transform.html#:~:text=If%20true%2C%20all%20the%20other%20indices%20will%20be%20removed%20from%20this%20alias%20before%20adding%20the%20destination%20index%20to%20this%20alias.%20Defaults%20to%20false). This requires [manage](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-privileges.html#:~:text=In%20that%20case%20the%20request%20requires%20the%20manage%20privilege%20as%20well%2C%20on%20both%20the%20index%20and%20the%20aliases%20names) privileges on the destination indices. 

Without the `manage` privileges, I got this error when installing transform containing aliases:  
```
Privileges check failed
Cannot create transform [logs-ti_otx.latest_ioc-default-0.1.0] because user elastic/kibana lacks the required permissions [logs-ti_otx.pulses_subscribed-*:[], logs-ti_otx_latest.destination-1:[manage], logs-ti_otx_latest.pulses_subscribed:[manage]]
```